pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -190,6 +193,45 @@ const (
 	KernelTunableModify KernelTunableBehavior = "modify"
 )
 
+// Init creates the top level qos cgroup containers
+// We currently create containers for the Burstable and Best Effort containers
+// All guaranteed pods are nested under the RootContainer by default
+// Init is called only once during kubelet bootstrapping.
+func InitQOS(qosConfig *QOSConfig) (QOSContainersInfo, error) {
+	cm := NewCgroupManagerImpl()
+	// The rootContainer under which all qos containers are brought up is configurable
+	// and can be specified through the --cgroup-root flag.
+	// We default to the system root / in case no root is specified
+	rootContainerName := defaultContainerRoot
```

Comment:
Cgroup-root default "" has been used at a lot places in the codebase. I thought I will once again confirm with you before I make it default to "/".